black/GameThing.h

```diff
@@ -36,128 +36,559 @@ struct Villager;
 
 enum GAME_THING_TYPE
 {
-  GAME_THING_TYPE_GFOOTPATH = 0x1,
-  GAME_THING_TYPE_GFOOTPATH_LINK = 0x2,
-  GAME_THING_TYPE_GFOOTPATH_NODE = 0x3,
-  GAME_THING_TYPE_GFOOTPATH_LINK_SAVE = 0x4,
-  GAME_THING_TYPE_GFOOTPATH_FINDER = 0x5,
-  GAME_THING_TYPE_SPELL_FOREST = 0xf,
-  GAME_THING_TYPE_CITADEL_HEART_1 = 0x3a,
-  GAME_THING_TYPE_CITADEL_HEART_2 = 0x3b,
-  GAME_THING_TYPE_POT = 0x58,
-  GAME_THING_TYPE_GGAME = 0x6a,
-  _GAME_THING_TYPE_COUNT = 0x6b
+  /* 0x000 */ GAME_THING_TYPE_UNUSED_000,
```

Comment:
The reversal of the values into comments will get reverted by the formatting script

Reply:
By reverted do you mean removed or it'll automatically add the values back to the right side? If it adds them back, then it shoudln't really matter? Either way the comments were just for clarity, even if they get removed I don't think it really matters too much.